### PR TITLE
Updated SDG Icons 

### PR DESCRIPTION
**Pull request by S-r-f-l**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,4 @@ represented by line or polygon geometries (e.g. protected areas or transimission
 
 ## More information on remote management methods
 The datamodel is part of KfWs RMMV Initiative. You can find more information [here](https://www.kfw-entwicklungsbank.de/Service/Publications-Videos/Publications-by-topic/Digitalisation/RMMV-Guidebook/). You can also find sample terms of reference there if you require external assistance on collecting the data. 
+test 2
```
Comment (fretchen) [inline README.md:22]: Could you please remove those ?

**Review by fretchen — APPROVED** [review]
This PR looks very nice. If you could just remove the small changes that you made to the README I am ok with this...

File: README.md (hunk 1 above)
Comment (S-r-f-l) [reply README.md]: Yes, i have removed it.

**Comment by Maja4Dev** [comment]
Since we are changing the location types, we will then also have to change and expand the corresponding location icons. Since there are a number of location types that can be assigned to more than one SDG, the sector economists requested to remove the SDG-connotation numbers from the icons and to only assign colours according to the most frequently used subsector. The „Generic/cross-sectoral“ one could be black or grey.

**Pull request merged**